iceoryx_hoofs/platform/win/source/mman.cpp

```diff
@@ -60,7 +60,7 @@ int iox_shm_open(const char* name, int oflag, mode_t mode)
                                 static_cast<LPSECURITY_ATTRIBUTES>(nullptr),
                                 static_cast<DWORD>(access),
                                 static_cast<DWORD>(MAXIMUM_SIZE_HIGH),
-                                static_cast<DWORD>(MAXIMUM_SIZE_LOW),
+                                static_cast<DWORD>(IOX_MAXIMUM_SUPPORTED_SHM_SIZE),
```

Comment:
Oh, this is not quite correct. You must put the high 32 bits of IOX_MAXIMUM_SUPPORTED_SHM_SIZE into MAXIMUM_SIZE_HIGH and the low 32 bits into MAXIMUM_SIZE_LOW ... it's called win32 for a reason :laughing: